TransportClient.js

```diff
@@ -13,7 +13,7 @@ class MQTTTransportClient {
         await this.mqttClient.subscribe(this.inTopic);
 
         this.mqttClient.on('message', (topic, data) => {
-            if (topic !== this.inTopic) return;
+            if (!this.inTopic.endsWith(topic)) return;
```

Comment:
Why do we need this change? This can introduce bugs to users.  If it is for $share prefix, then we can introduce bugs with $share prefix as well. As one client will send request but the respone will be received by an another client.

Reply:
Will need to check tests - for incoming $share isn't part of the topic, only outgoing.